gcloud/bigtable/cluster.py

```diff
@@ -317,11 +206,12 @@ def create(self):
         """
         request_pb = _prepare_create_request(self)
         # We expect a `google.longrunning.operations_pb2.Operation`.
-        operation_pb = self._instance._client._instance_stub.CreateCluster(
-            request_pb)
+        client = self._instance._client
+        operation_pb = client._instance_stub.CreateCluster(request_pb)
 
-        op_id = _process_operation(operation_pb)
-        return Operation('create', op_id, cluster=self)
+        operation = Operation.from_pb(operation_pb, client)
+        operation.target = self
+        return operation
```

Comment:
Yes, I was deliberately discarding that information:  the `metadata` argument to the constructor would allow a place to keep it, if desired (`from_pb` would need to pass through extra information for that to work).

The `type_url` of the protobuf metadata (which *is* preserved by `from_pb` if the `type_url` is registered) would indicate that the operation was returned from a `CreateInstance` request.